integrations/nielsen-dcr/lib/index.js

```diff
@@ -165,18 +165,19 @@ NielsenDCR.prototype.heartbeat = function(assetId, position, options) {
 
 NielsenDCR.prototype.getContentMetadata = function(track, type) {
   var properties = 'properties.';
-  var assetIdProp = 'asset_id';
-  if (type) {
-    if (type === 'preroll') properties = 'properties.content.';
-    if (type === 'video') assetIdProp = 'content_asset_id';
-  }
-
+  if (type && type === 'preroll') properties = 'properties.content.';
+
+  var customAssetId = this.options.contentAssetIdPropertyName
```

Comment:
I think it might be more clear to just use a single conditional here instead of the ternary operator:

```
var customAssetId;
if (this.options.contentAssedItPropertyName) {
    customAssetId = track.proxy(properties + this.options.contentAssetIdPropertyName)
}